apps/patient/src/views/Main.tsx

```diff
@@ -63,6 +64,9 @@ export const Main = () => {
 
     setFlattenedFills(countFillsAndRemoveDuplicates(order.fills));
 
+    datadogRum.addRumGlobalContext('organizationId', order.organization.id);
```

Comment:
I just added updated datadog, so this method doesn't exist any more, it should be https://github.com/Photon-Health/client/blob/5c1e05d0c9b7f8687e01e504073ad38362ca8650/apps/app/src/views/routes/Main.tsx#L45


https://docs.datadoghq.com/real_user_monitoring/browser/advanced_configuration/?tab=npm#global-context